plex_cli.py

```
@@ -995,6 +998,166 @@ def handle_addon_command(args: List[str]) -> int:
+            port_input = input("Could not detect port from config. Enter port: ").strip()
+            if port_input.isdigit():
+                port = int(port_input)
+                if 1 <= port <= 65535:
+                    break
+                else:
+                    print_error("Port must be between 1 and 65535.")
+            else:
+                print_error("Port must be a number.")
+    else:
+        print_info(f"Detected port: {port}")
+        port_input = input(f"Use this port? (Y/n, or enter a different port): ").strip()
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)